docs/.vuepress/config.js

```diff
@@ -84,7 +84,8 @@ module.exports = {
           ['/guides/overview', 'Overview'],
           ['/guides/slp-implementation-instructions', 'Implementation Instructions'],
           ['/guides/create-slp-faucet', 'Create your own SLP Faucet'],
-	  ['/guides/add-slp-icon', 'Add an icon for your SLP Token'],    
+		  ['/guides/add-slp-icon', 'Add an icon for your SLP Token'],
```

Comment:
Indentation is off

Reply:
Ugh, spaces vs tabs, looked fine on my machine. Fixed.